extensions/git/src/git.ts

```diff
@@ -550,8 +550,8 @@ export class Git {
 
 		if (options.log !== false) {
 			const startTime = Date.now();
-			child.on('exit', (_) => {
-				this.log(`> git ${args.join(' ')} [${Date.now() - startTime}ms]\n`);
+			child.on('exit', (code) => {
+				this.log(`> git ${args.join(' ')} [${Date.now() - startTime}ms]${code === null ? ' (cancelled)' : ''}\n`);
```

Comment:
Not sure the following assertion is true: if the code is `null`, the is means that the child was killed by us. There could be other reasons for the code to be `null`. There could also be other reasons for the child to be killed.

Reply:
I have switched to using the `killed` property. Another idea that I considered was to pass a custom signal when killing the process as part of cancellation but if I read the documentation correctly, on Windows the signal is being ignored so I am not sure that we would be able to access that. Let me know if you have a better suggestion. Thanks!